cask-cli.el

```diff
@@ -260,8 +263,14 @@ If no files directive or no files, do nothing."
       (princ (concat file "\n")))))
 
 (defun cask-cli/build ()
-  "Build all Elisp files in the files directive."
-  (cask-build (cask-cli--bundle)))
+  "Build all Elisp files in the files directive.
+
+Exit code is 1 if any of the files contained a compile error, 0
+otherwise."
+  (let ((result (cask-build (cask-cli--bundle) :warn-as-error cask-cli--warn-as-error)))
+    (if result
+        (kill-emacs 0)
+      (kill-emacs 1))))
```

Comment:
What's `result` here?  Which type is it?

And please do `(kill-emacs (if result 0 1))`…